assets/benchmarks/shaders/Benchmark_Texture.hlsl

```diff
@@ -15,8 +15,42 @@
 #include "Benchmark_Quad.hlsli"
 
 Texture2D Tex0 : register(t1);  // Slot 0 is used by push constant.
```

Comment:
i am wondering if you could use Texture2D tex[10] so that it would be easier to extend in the future?